features/telemetry/protocols/sflow/parser/src/main/java/org/opennms/netmgt/telemetry/protocols/sflow/parser/SFlowUdpParser.java

```diff
@@ -179,6 +179,16 @@ public String getName() {
         return name;
     }
 
+    @Override
+    public String getDescription() {
+        return "SFlow";
+    }
+
+    @Override
+    public Object dumpInternalState() {
+        return null;
```

Comment:
Can you create issues for the unimplemented methods for SFlow, BMP, and so on?

Reply:
There is no internal state for SFlow but I've added a minimal state for BMP.

Reply:
And added a comment to the SFlow implementation.